src/renderer/components/chart/bar-chart.tsx

```diff
@@ -136,7 +136,8 @@ export class BarChart extends React.Component<Props> {
       },
       plugins: {
         ZebraStripes: {
-          stripeColor: chartStripesColor
+          stripeColor: chartStripesColor,
+          interval: chartData.datasets.length ? chartData.datasets[0].data.length : 61
```

Comment:
I see that `61` is used in other places too. However it seems like a strange number. Maybe add a comment about why it was chosen?

Reply:
Yes, you're right. Actually, we don't need any predefined numbers in here since `datasets[0].data.length` always contains needed value. Fixed.